buildpack.md

```diff
@@ -770,17 +777,21 @@ The stack `build-images` and `run-images` are suggested sources of images for pl
 [[processes]]
 type = "<process type>"
 command = "<command>"
+args = ["<arguments>"]
 
 [[slices]]
 paths = ["<app sub-path glob>"]
 ```
 
 The buildpack MAY specify any number of processes or slices.
 
-For each process, the buildpack MUST specify:
+For each process, the buildpack:
 
-- A unique process type for each entry within a `launch.toml` file.
-- A command that is valid when executed using the Bash 3+ shell.
+- MUST specify a `type` that is not identical to other process types provided by the same buildpack.
+- MUST specify a `command` that is either:
+  - A command sequence that is valid when executed using the Bash 3+ shell, if `args` is not specified.
+  - A path to an executable or the file name of an executable in `$PATH`, if `args` is a list with zero or more elements.
```

Comment:
So we might see stuff like this right?

```
[[processes]]
type = "web"
command = "myapp"
args = ["--"]
```

Reply:
More like:
```toml
[[processes]]
type = "web"
command = "myapp"
args = []
```
To force bashless mode.

Reply:
The complexity arises from the different cases:

For Bash:
Valid, common: `<shell-command arg1 arg2>`
Valid, uncommon: `<binary> <arg1> <arg2>`
Valid, uncommon: `<path-to-binary> <arg1> <arg2>`
Valid, but very confusing: `<shell-command> <arg1> <arg2>`

For direct exec:
Invalid: `<shell-command> <arg1> <arg2>`
Invalid: `<shell-command arg1 arg2>`
Valid: `<binary> <arg1> <arg2>`
Valid: `<path-to-binary> <arg1> <arg2>`